### PR TITLE
docs/lois

### PR DESCRIPTION
**Pull request by zmzlois**
Update

vscode: 
- based on extension updates
- add SSH connection

Added
- jetbrains
- idps

### CODE REVIEW

File: src/content/docs/installation/idps.mdx [hunk 1]
```diff
@@ -0,0 +1,544 @@
...
+description: How to add identity providers from Github, Bitbucket and Gitlab for your Daytona installation on Keycloak 
+sidebar:
+  label: IDPs
+---
+
+import Icon from "../../../components/Icons.astro"
+import Table from "../../../components/Table/Table.astro"
+import Keyboard from "../../../components/Keyboard.astro"
+
+{/*TODO: All these shit should be just a link click away to set up no? Related excalidraw: https://link.excalidraw.com/l/7LItosJ86GN/2bnFDQd3mNq*/}
```
Comment (nkkko) [inline src/content/docs/installation/idps.mdx:12]: Could you please avoid using inappropriate words, even if it is just a comment for yourself? :D Tnx

File: src/components/Keyboard.astro
Comment (nkkko) [inline src/components/Keyboard.astro:1]: Please avoid adding new components and focus on the docs themself. Any need for new components should be directed to external dev team.

**Review by nkkko — CHANGES_REQUESTED** [review]
We appreciate the effort, it is clear that you did a lot of work. But it makes it hard to coordinate considering the number of people involved.

Could you please:
- Please clean up and just do a PR with plain docs change.
- Any code change, new components etc. should be done in stages, for example, propose a new components in a separate PR which can then be reviewed by external devs. Only when new components are merged add them to doc files.

**Comment by zmzlois** [comment]
closing this to split style + component + plain docs

**Pull request closed without merging**